### ISSUE TITLE
The name, role, state, and value of all user interface components can be programmatically determined (especially when the state changes).

### ISSUE THREAD

**Issue by justpie** [Purdue/purdue-wp-theme#145]
The social media "Facebook" through "LinkedIn" links have no accessible names. Ensure topProvide an accessible name to these links, such as "Facebook".

The social media links, such as "Facebook", "Twitter", "LinkedIn" which are not visible underneath "Learn More" button in the "A Tradition...Possibility" heading open in new windows without informing screen-reader users of this beforehand. Add an aria-label to each that includes the visible link text and "opens in a new window", or change the visible link text to inform users that each link opens in a new window. Alternatively, have the links open on the same page so a warning is no longer necessary.
